TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PowerPlay/PowerPlayAutoRoadRunner.java

```diff
@@ -53,8 +53,8 @@ public class PowerPlayAutoRoadRunner extends LinearOpMode {
     // options framework handling. Initialization may be too early, in which case move into setup
     private OptionsFile options;
     private MatchTimer timer = new MatchTimer(30000);
-    private Pose2d START_POS_RIGHT = new Pose2d(34, -62, Math.toRadians(90));
-    private Pose2d START_POS_LEFT = new Pose2d(-34, -62, Math.toRadians(90));
+    public static Pose2d START_POS_RIGHT = new Pose2d(34, -62, Math.toRadians(90));
+    public static Pose2d START_POS_LEFT = new Pose2d(-34, -62, Math.toRadians(90));
     boolean doScoring;
     int numTimesToCycle;
     TrajectorySequence trajectoryOne;
```

Comment:
Why are the other trajectories private and this one isn't?  (Not related to this PR)